kubernetes/config/kube_config.c

```diff
@@ -0,0 +1,309 @@
+#include "kube_config.h"
+#include "kube_config_yaml.h"
+#include <errno.h>
+#include <unistd.h>
+
+#define ENV_KUBECONFIGE "KUBECONFIG"
```

Comment:
`ENV_KUBECONFIG` ? (why the trailing `E`)

Reply:
Typo. I will fix it.